### DISCUSSION TITLE
What does the load() function expect as a return when we are adding a new storage?

### DISCUSSION THREAD

**Discussion by capoia** [GrapesJS/grapesjs#5298] [Q&A] [answered]
Hello, I would like to know what exactly I need (the format) to return in the load() function for it to properly load the components and styles.
I apologize if this is somewhere, I really looked for it even within the repository, but I only found a generic type:
![CleanShot 2023-08-10 at 18 27 12@2x](https://github.com/GrapesJS/grapesjs/assets/21159928/9b0c6152-ea27-473e-b6a2-14f6b9c87d52)

**Comment by artf** [comment, marked as answer]
The `ProjectData`, is the GrapesJS JSON of the project, eg. the one you get with `editor.getProjectData()`
